clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -21,6 +21,7 @@ TODO(krzysied): This measurement should replace api_responsiveness.go.
 package slos
 
 import (
+	"flag"
```

Comment:
nit: "github.com/spf13/pflag" is used by clusterloader. The "flag" will work, but I would suggest using "pflag" for consistency.